qualtran/bloqs/data_loading/qroam_clean.py

```diff
@@ -205,6 +205,94 @@ def wire_symbol(self, reg: Optional[Register], idx: Tuple[int, ...] = tuple()) -
         raise ValueError(f'Unknown register name {name}')
 
 
+@attrs.frozen
+class QROAMCleanAdjointWrapper(Bloq):
```

Comment:
Should we keep this as QROAMCleanAdjoint and rename QROAMCleanAdjoint TableLookupAdjoint (or something appropriate)

Reply:
I thought about it and I think `TableLookupAdjoint` would be too general, especially as we add more adjoint bloqs that have different tradeoffs for using dirty vs clean ancillas; along with using measurement based uncomputation. 

the current name is closely tied to its implementation; so I think it's fine to keep it as is.